docs/book/src/user/quick-start.md

```diff
@@ -731,7 +731,7 @@ spec:
 {{#/tab }}
 {{#/tabs }}
 
-After the controlplane is up and running, let's retrieve the [target cluster] Kubeconfig:
+After the controlplane is up and running, let's retrieve the [target management cluster] Kubeconfig:
```

Comment:
This is actually the workload cluster. But I see Workload cluster is missing from the glossary.

If you change this to 
```suggestion
After the controlplane is up and running, let's retrieve the [workload cluster] Kubeconfig:
```

And add the Workload cluster definition to the glossary and link to it then we'll be good to get this merged